nncf/experimental/post_training/algorithms/quantization/parameters.py

```diff
@@ -0,0 +1,80 @@
+from typing import List
+from typing import Optional
+
+from nncf.common.utils.ordered_enum import OrderedEnum
+from nncf.common.quantization.structs import QuantizerConfig
+from nncf.common.quantization.structs import QuantizationMode
+
+from nncf.experimental.post_training.algorithms import AlgorithmParameters
+from nncf.experimental.post_training.algorithms import PostTrainingAlgorithms
+
+from nncf.experimental.post_training.algorithms.quantizer_range_finder import QuantizerRangeFinderParameters
+
+
+class PRESET(OrderedEnum):
+    PERFOMANCE = 'perfomance'
+    MIXED = 'mixed'
+    ACCURACY = 'accuracy'
+
+
+class GRANULARITY(OrderedEnum):
+    PERTENSOR = 'pertensor'
+    PERCHANNEL = 'perchannel'
+
+
+class PostTrainingQuantizationParameters(AlgorithmParameters):
+    def __init__(self,
+                 preset: PRESET = PRESET.MIXED,
+                 weight_bits: int = 8,
+                 weight_granularity: GRANULARITY = GRANULARITY.PERCHANNEL,
+                 activation_bits: int = 8,
+                 activation_granularity: GRANULARITY = GRANULARITY.PERTENSOR,
+                 range_type: str = 'min_max',
```

Comment:
It will be better if this is defined as a class not str

Reply:
This is actually an open question - what type of arguments are more user-friendly.
There are two options:

1. Using string
2. Using Enum

Advantages and disadvantages using strings and otherwise for using Enum:
**+** : User doesn't need to import the Enum inherited class to pass the argument.
**-** : User has to carefully read the docs to get the correspondence between strings and arguments.

Reply:
There is the ticket (Ref: 79918) for this problem.

Reply:
But I think that @asenina comment is valid. You use different styles to pass parameters. For example, for the `present` parameter you use `Enum`, but for the `range_type` parameter you use `str`. It seems strange for me. I think it is better to use only one approach for this.

Reply:
I agree with you. Style should be the same. I just don't know what solution should be chosen.

Reply:
Translate string parameters to Enum